features/test/test-service.feature

```diff
@@ -8,14 +8,11 @@ Feature: testing an Exosphere service
   - when running "exo test" in a service directory, it runs the test for the respective service
 
   Scenario: testing a service
-    Given I am in the "exoservice" directory of the "tests-cucumber" example application
-    When starting "exo test" in my service directory
-    Then it prints "Testing service 'exoservice'" in the terminal
-    And I eventually see:
-    """
-    1 scenario (1 passed)
-    """
-    And it prints "'exoservice' tests passed" in the terminal
+    Given I am in the root directory of the "tests-passing" example application
+    When starting "exo test" in the "tweets-service" directory
+    Then it prints "Testing service 'tweets-service'" in the terminal
+    And it prints "'tweets-service' tests passed" in the terminal
+    And it does not print "'users-service' tests passed" in the terminal
```

Comment:
Thoughts on having this be `Testing service 'users-service'` instead of `'users-service' tests passed` as the latter wouldn't be printed if the user tests failed